src/CommunityToolkit/Xamarin.CommunityToolkit/Xamarin.CommunityToolkit.csproj

```diff
@@ -64,8 +64,8 @@
     <Compile Include="**/*.shared.*.cs" />    
     <None Include="..\..\..\LICENSE" PackagePath="" Pack="true" />
     <None Include="..\..\..\assets\XamarinCommunityToolkit_128x128.png" PackagePath="icon.png" Pack="true" />
-    <PackageReference Include="Xamarin.Forms" Version="5.0.0.2125" />
```

Comment:
Let's not update this NuGet Package unless we have to, because it'll force all Xamarin.CommunityToolkit users to then upgrade to the latest version Xamarin.Forms